client/src/components/DisplayAvatar.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Avatar from "@material-ui/core/Avatar";
+import { makeStyles } from "@material-ui/core/styles";
+
+const DisplayAvatar = ({ user, width, height }) => {
+	const useStyles = makeStyles((theme) => ({
+		root: {
+			display: "flex",
+			"& > *": {
+				margin: theme.spacing(1)
+			}
+		},
+		avatar: {
+			width: theme.spacing(width),
+			height: theme.spacing(height),
+			borderRadius: "50px"
+		}
+	}));
+
+	const classes = useStyles();
+
+	let avatarCard = user.avatar ? (
+		<img
+			className={classes.avatar}
+			src={`${window.location.origin}/api/users/avatar/${user._id}`}
```

Comment:
I don't think you need window.location.origin if it's the same domain

```suggestion
			src={`/api/users/avatar/${user._id}`}
```